server/data/converters/serviceArea.test.ts

```diff
@@ -0,0 +1,74 @@
+import { ServiceAreaListResponseDataItem } from '../strapiApiTypes'
+import { convertServiceArea } from './serviceArea'
+
+test('check conversion', () => {
```

Comment:
This test demos the difference between the model response (that's we'd expose to the rest of the app) and the strapi response.

Also exposing a test factory data method for use elsewhere to reduce coupling of the data structure to the rest of the tests.